package.json

```diff
@@ -54,6 +54,7 @@
     "@now/node": "^1.6.1",
     "@philpl/buble": "^0.19.7",
     "babel-preset-gatsby": "^0.2.29",
+    "carbon-components-react": "^7.16.0",
```

Comment:
Should we `carbon-components`, as well? Not sure where it went, seems like we might be using the hoisted version from `gatsby-theme-carbon` so might help to add it in here, too.

Reply:
removed. :)